### PR TITLE
refactor: break up consulting, remove unused description field, move out unrelated

### PR DESCRIPTION
**Pull request by mirestrepo**
This also touches other files ins services with unused description field

### CODE REVIEW

File: about/teams.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+### Computational Biology Core
+Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-499a-a1f3-5f9b24c390e4)
+>The COBRE-CBHD Computational Biology Core (CBC) integrates across multiple research focused Centers supporting data-intensive research using high-throughput DNA/RNA sequencing datasets. CBC provides help for experimental design and data processing pipelines for high-throughput datasets generated in individual projects, particularly for DNA/RNA sequencing data.
+
+### High-Performance Computing
+Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-499a-a1f3-5f9b24c390e4)
+>Our software engineers help researchers to get the most out of high-performance computing (HPC) resources. To achieve this, we partner with users to design, debug, and optimize their code to run in an HPC environment. We also collaborate with new and non-traditional HPC users to help them to scale up and accelerate their science.
+
+### GSDC (Graphics, Software and Data Core)
```
Comment (mcmcgrath13) [inline about/teams.md:14]: this should also have sections for hpc system and research technical services

File: schemas/services-frontmatter.yml [hunk 2]
```diff
@@ -1,8 +1,2 @@
 title: str()
```
Comment (mcmcgrath13) [inline schemas/services-frontmatter.yml:1]: is `links` not used anymore? I thought that was still in use (though I could definitely be misremembering)

**Review by mcmcgrath13 — APPROVED** [review]
🙌

File: about/teams.md (hunk 1 above)
Comment (mirestrepo) [reply about/teams.md:14]: absolutely! I'm getting some blurbs from Mete, but I'll merge this first

File: schemas/services-frontmatter.yml (hunk 2 above)
Comment (mirestrepo) [reply schemas/services-frontmatter.yml:1]: I just checked again and not in the frontmatter, but yes in the main category, which is still in place

**Pull request merged**